bailey/bailey.py

```diff
@@ -94,17 +94,20 @@
 grammar = Grammar(
     r"""
     expr       = (entry / emptyline )*
-    entry      = hash headword comma pos ws senses subentry emptyline
+    entry      = hash headword comma pos ws senses subentry period emptyline
+    # entry      = hash headphrase comma pos ws senses subentry period emptyline
     hash       = (~"#")*
+    # headphrase = headword (ws headword)*
```

Comment:
I've added this to capture the head words with multiple words. Most of the exception are due to this.

However I couldn't get this to work.